pom.xml

```diff
@@ -186,11 +186,12 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <mojo.java.target>1.8</mojo.java.target>
+    <mojo.java.target>8</mojo.java.target>
     <maven.compiler.source>${mojo.java.target}</maven.compiler.source>
     <maven.compiler.target>${mojo.java.target}</maven.compiler.target>
 
-    <minimalJavaBuildVersion>${mojo.java.target}</minimalJavaBuildVersion>
+    <!-- as expected by Enforcer -->
+    <minimalJavaBuildVersion>1.8</minimalJavaBuildVersion>
```

Comment:
It a little will complicate when project set target as 8 also want to check if used JDK is ok. 
In such case we will must remember to set another property also.

My proposition is to postpone this, and allow 8 as 1.8  for maven enforcer.

Reply:
https://issues.apache.org/jira/browse/MENFORCER-440

Reply:
enforcer released and upgraded here, we can continue with it